backend/worker_handler/utils_worker.py

```diff
@@ -0,0 +1,71 @@
+
```

Comment:
I think you could rename this file to worker_aux_functions, as we are using that, or even change the order, worker_utils makes more sense.